client/my-sites/stats/overview.jsx

```diff
@@ -88,21 +88,22 @@ export default React.createClass( {
 			);
 
 			return overview;
-		}, this );
+		} );
 
 		return (
-			<Main wideLayout={ true }>
+			<Main wideLayout>
+				<QuerySites allSites />
```

Comment:
Indeed there aren't many flows which land one on the overview page, but it does happen.  Based on the conversation, it appears we might need to roll back this PR, or at least add in the observe mixin again ( gasp! ) until sites list is migrated.